lib/avalon/m3u8_reader.rb

```diff
@@ -24,7 +24,7 @@ def self.read(io, recursive: true)
         new(io.read, recursive: recursive)
       elsif io.is_a?(String)
         if io =~ /^https?:/
-          URI.open(io, "Referer" => Rails.application.routes.url_helpers.root_url) { |resp| new(resp, Addressable::URI.parse(io), recursive: recursive) }
+          URI.open(io, "Referer" => Settings.streaming.auth_referer || Rails.application.routes.url_helpers.root_url) { |resp| new(resp, Addressable::URI.parse(io), recursive: recursive) }
```

Comment:
Should this param be added (maybe commented out?) to the `settings.yml` file so that implementers know it is available without having to view the external docs?

Reply:
@masaball I added this.  Could you please re-review?